docs/src/tutorial/simple_system.md

```diff
@@ -234,23 +301,30 @@ If you are not sure whether you have the latest version, please upgrade to ensur
     *0.8*, respectively. It should like the image below.
 ![image](../figs_simple_system/simple_system_fix_ratio_out_in_unit_flow.png)
 
+!!! note
+    The order of the nodes is important for the *fix\_ratio\_out\_in\_unit\_flow* 
+    parameter. If you have swapped the nodes or inverted the efficiency values, 
+    the Run SpineOpt tool will run into errors.
+
 When you're ready, commit all changes to the database.
```

Comment:
Assume this is the same steps as saving.

Reply:
Also here I've made it more explicit.